javascript/main.js

```diff
@@ -19,21 +19,34 @@ fetch('https://www.thecocktaildb.com/api/json/v1/1/search.php?s=margarita')
 
       const img = document.createElement('img');
       img.setAttribute('src',`${r.strDrinkThumb}`);
+      img.classList.add("images")
 
-      const h1 = document.createElement("h1");
+      const h1 = document.createElement("h3");
       h1.textContent ='ID: '+ r.idDrink;
-      const p = document.createElement("p");
-      p.textContent = r.strInstructions; 
-
+      
       const innerDiv=document.createElement('div');
       innerDiv.setAttribute('class','innerDiv');
 
+      const drinkName=document.createElement("h3");
+      drinkName.classList.add("drinkName")
+      drinkName.textContent=" Drink name :"+r.strDrink;
+
+      const alcoholic =document.createElement("h3")
+      alcoholic.textContent="Alcoholic :"+r.strAlcoholic
+
+
+      innerDiv.addEventListener("click",function(){
+      const p = document.createElement("p");
+      p.textContent ="Instructions :"+r.strInstructions;   
+       innerDiv.appendChild(p);
+      })
+      innerDiv.appendChild(drinkName);
       innerDiv.appendChild(h1);
-      innerDiv.appendChild(p);
+   innerDiv.appendChild(alcoholic)
 
       content.appendChild(img);
       content.appendChild(innerDiv);
       drinks.appendChild(content);
 
     };
-  });
\ No newline at end of file
+  }).catch((eror)=>console.log(eror))
```

Comment:
when I test your code 

![Screenshot from 2021-07-29 14-04-10](https://user-images.githubusercontent.com/47992412/127481381-dd77f3b4-0923-4418-8dd2-71ae4ac7c612.png)

Reply:
we push it after we test catch ang forget edit it